src/mw-backbone/model/mw_model.js

```diff
@@ -1,17 +1,15 @@
 mwUI.Backbone.Model = mwUI.Backbone.NestedModel.extend({
   selectable: true,
-  basePath: '',
+  hostName: function(){
+    return mwUI.Backbone.hostName;
+  },
+  basePath: function(){
```

Comment:
They are used by the `getUrl` method. The `getUrl` method is used by the collection method `url` and the model method `urlRoot`. They can be either a string or a function and by default they return the value that is globally defined by setting the vars in the `mwUI.Backbone`object